output/cloud/expv2/mapping.go

```diff
@@ -10,15 +10,12 @@ import (
 )
 
 // TODO: unit test
```

Comment:
Reminder: It makes sense to address it here

Reply:
https://github.com/grafana/k6/pull/3162 changes the API, better to make it after we merge it.